src/App.jsx

```diff
@@ -14,33 +15,93 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_FIELD_LENGTH = 'length';
+const SORT_FIELD_ALPHABET = 'alphabet';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function getVisibleGoods(goods, { sortField, reverse }) {
+  const visibleGoods = [...goods];
```

Comment:
[Not critical, just for reference]
Instead of copying an array to use mutating `sort` method, we can use non-mutating `toSorted` method as alternative